services/src/main/java/org/keycloak/protocol/oidc/mappers/OIDCAttributeMapperHelper.java

```diff
@@ -144,6 +153,18 @@ private static Boolean getBoolean(Object attributeValue) {
         if (attributeValue instanceof String) return Boolean.valueOf((String) attributeValue);
         return null;
     }
+    
+    private static JsonNode getJsonNode(Object attributeValue) {
+        if (attributeValue instanceof JsonNode) return (JsonNode) attributeValue;
+        if (attributeValue instanceof String) {
+            try {
+                final ObjectMapper mapper = new ObjectMapper();
```

Comment:
You should consider using `org.keycloak.util.JsonSerialization` instead of creating a new `ObjectMapper` instance.

Don't you want to throw an exception in case the `attribute` value is not a `JsonNode`. Why you need to convert the string to a json object ?